pkg/operator/ceph/csi/spec.go

```diff
@@ -177,7 +172,8 @@ func ValidateCSIParam() error {
 	return nil
 }
 
-func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *version.Info) error {
+// StartCSIDrivers ...
```

Comment:
the function comment doesn't make any sense

Reply:
Yeah I'll remove it.